spec/displayer_spec.rb

```diff
@@ -14,15 +14,13 @@ def gets
 end
 
 describe "Displayer" do
-  before(:each) do
-    @display = Displayer.new(TestOutputFunction.new, TestInputFunction.new)
-  end
+  subject(:display) { Displayer.new(TestOutputFunction.new, TestInputFunction.new) }
+
   it "applies the output function to the message" do
```

Comment:
I feel like the test description of what the `Displayer` does is a little bit too _technical_. Tests are used as documentation, as well, and it would be nice to have a description that sounds less technical/implementation focused and more of a natural description.

Ie, I'd rename `applies the output function to the message` to: `outputs the given message`
and
`applies the input function to receive player input` to: `gets the players input`

That way if you end up changing the method `output` or `input`, the test description can still accurately describe what's happening.

Reply:
Ok, I see what you mean. I thought it was better to be more specific, but if the code changes later I can see how that would get confusing.